connect/models/tier_config.py

```diff
@@ -95,6 +95,36 @@ class TierConfig(BaseModel):
     open_request = None  # type: Optional[BaseModel]
     """ (:py:class:`.BaseModel` | None) Reference to TCR. """
 
+    @classmethod
+    def get(cls, tier_id, product_id, config=None):
+        """
+        Gets the specified tier config data. For example, to get Tier 1 configuration data
+        for one request we can do: ::
+
+            TierConfig.get(request.asset.tiers.tier1.id, request.asset.product.id)
+
+        :param str tier_id: Id of the requested Tier Config.
+        :param str product_id: Id of the product.
+        :param Config config: Config to use, or ``None`` to use environment config (default).
+        :return: The requested Tier Config, or ``None`` if it was not found.
+        :rtype: Optional[TierConfig]
+        """
+        from connect.resources.base import ApiClient
+
+        response, _ = ApiClient(config, base_path='tier/config-requests').get(
+            params={
+                'status': 'approved',
+                'configuration__product__id': product_id,
```

Comment:
Frankly speaking double-underscore notation is unofficial and will be not supported in future 
right notation is dots: configuration.product.id

Reply:
OK, I'll change it. Apart from these, FulfillmentAutomation passes the list of products in the 'asset.product.id__in' query param. I assume it should be 'asset.product.id.in' too, right?

Reply:
not for the __in

Reply:
Ok, fixed.